src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -0,0 +1,152 @@
+            checkbox.removeAttribute('checked');
+            checkbox.checked = false;
+        });
+        sectionSelect.querySelector('option').selected = true;
+        lastModifiedSelect.querySelector('option').selected = true;
+        doc.querySelector(lastModifiedModal.dataset.periodSelector).value = '';
+        doc.querySelector(lastModifiedModal.dataset.endSelector).value = '';
+        lastCreatedSelect.querySelector('option').selected = true;
+        doc.querySelector(lastCreatedModal.dataset.periodSelector).value = '';
+        doc.querySelector(lastCreatedModal.dataset.endSelector).value = '';
+    };
+    const toggleApplyBtnState = () => {
+        const contentTypeOption = contentTypeSelect.querySelector('option');
+        const isContentTypeSelected = contentTypeOption.innerHTML !== contentTypeOption.dataset.default;
+        const isSectionSelected = !!sectionSelect.value;
+        const isModifiedSelected = !!lastModifiedSelect.value;
+        const isCreatedSelected = !!lastCreatedSelect.value;
+        const isEnabled = isContentTypeSelected || isSectionSelected || isModifiedSelected || isCreatedSelected;
+        const methodName = isEnabled ? 'removeAttribute' : 'setAttribute';
+
+        applyBtn[methodName]('disabled', !isEnabled);
+    };
+    const toggleFilters = (event) => {
+        event.preventDefault();
+
+        doc.querySelector('.ez-filters').classList.toggle('ez-filters--collapsed');
+    };
+    const toggleContentTypeSelector = (event) => {
+        event.preventDefault();
+
+        doc.querySelector('.ez-content-type-selector').classList.toggle('ez-content-type-selector--collapsed');
+    };
+    const toggleModal = (event) => {
+        const modal = $(event.target.dataset.targetSelector);
+
+        if (event.target.value !== 'custom_range') {
+            doc.querySelector(modal[0].dataset.periodSelector).value = event.target.value;
+            doc.querySelector(modal[0].dataset.endSelector).value = '';
+
+            toggleApplyBtnState();
+
+            return;
+        }
+
+        modal.modal('show');
+    };
+    const toggleGroup = (event) => {
+        event.preventDefault();
+
+        event.currentTarget.closest('.ez-content-type-selector__group').classList.toggle('ez-content-type-selector__group--collapsed');
+    };
+    const filterByContentType = () => {
+        const selectedCheckboxes = contentTypeCheckbox.filter(checkbox => checkbox.checked);
+        const contentTypesText = selectedCheckboxes.map(checkbox => checkbox.value).join();
+        const option = contentTypeSelect.querySelector('option');
+        const defaultText = option.dataset.default;
+
+        option.innerHTML = contentTypesText || defaultText;
+
+        toggleApplyBtnState();
+    };
+    const dateFields = [...doc.querySelectorAll('.ez-date-select')];
+    const dateConfig = {
+        formatDate: (date) => (new Date(date)).toLocaleDateString()
+    };
+    const checkAllFieldsFilled = (modal) => {
```

Comment:
I believe it checks only whether all select fields are filled, not all (like every kind of fields).